sdk/python/flet/tooltip.py

```diff
@@ -84,10 +85,10 @@ def __init__(
         gradient: Optional[Gradient] = None,
         border: Optional[Border] = None,
         border_radius: BorderRadiusValue = None,
-        shape: Optional[BoxShape] = None,
+        shape: Union[BoxShape, BoxShapeString] = None,
         message: Optional[str] = None,
         text_style: Optional[TextStyle] = None,
-        text_align: TextAlign = TextAlign.NONE,
+        text_align: Union[TextAlign, TextAlignString] = TextAlign.NONE,
```

Comment:
Let's make it `None` as everything else?

Reply:
I added `None` to the `TextAlignString` Literal.
Is that what you meant?